### DISCUSSION TITLE
F1 configuration not saved for next boot

### DISCUSSION THREAD

**Discussion by yoshinoyatoko** [memtest86plus/memtest86plus#385] [Enhancement]
Hi, Everyone:
It seems user's configuration not saved for next time.
For example, user change some setting with F1 hot key, only function in this runtime.

Is there any plan to save these settings?

I have no idea for using what nvram, CMOS available area?

**Comment by debrouxl** [comment]
That's correct, memtest86+ configuration is not saved. Probably partially because AFAIK, there's no portable way to store such information.
Note that memtest86+ is partially scriptable through the command line; I created some tracking issues for further scriptability, e.g. selecting tests.

**Comment by yoshinoyatoko** [comment]
Got it, thanks!
